shared/build.gradle.kts

```diff
@@ -86,30 +78,22 @@ kotlin {
         }
 
     sourceSets["androidMain"].dependencies {
-        implementation(kotlin("stdlib", Versions.kotlin))
-        implementation(Deps.SqlDelight.driverAndroid)
-        implementation(Deps.Coroutines.android)
-        implementation(Deps.Ktor.androidCore)
+        implementation(libs.sqlDelight.android)
+        implementation(libs.ktor.client.okHttp)
     }
 
     sourceSets["androidTest"].dependencies {
-        implementation(Deps.KotlinTest.jvm)
-        implementation(Deps.KotlinTest.junit)
-        implementation(Deps.AndroidXTest.core)
-        implementation(Deps.AndroidXTest.junit)
-        implementation(Deps.AndroidXTest.runner)
-        implementation(Deps.AndroidXTest.rules)
-        implementation(Deps.Coroutines.test)
-        implementation(Deps.robolectric)
+        implementation(libs.bundles.shared.androidTest)
```

Comment:
I'm curious where we'll decide to split the dependancies. For example you have all the android test dependencies in one bundle but then in iOSMain the dependancies are separated. Neither is wrong but just different organizations.

Reply:
Yes, I thought about this.

I decided to bundle for test as those dependencies doesn't affect the actual binary, so we can just add anything to test bundle and we're okay.

I didn't do for `iosMain` because those are separate dependencies and someone would care about looking at them separately and understand what we're using.

It would still bundle things under `main` where they're from same category. For example, if we're using `sqlDelight` and `sqlCipher`, I would bundle them as `commonDb`. I did the same for bundling all `ui` dependencies under one bundle for the android app.

This is what I have thought so far. Let me know if you have any feedback on this approach.

Reply:
That sounds right to me, yea combining based on category makes sense.